test/xref2/module_type_of_size.t/compile.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
```

Comment:
That's not a portable `#!` line. I suggest not having one and to run this script using `bash ./compile.sh`.

Reply:
Huh? I don't understand - I think this script is pretty portable. The `gen.sh` needs to be changed to `#!/bin/bash` though!

Reply:
it doesn't use /bin/bash, it's /bin/sh - does that not work on your machine?